### DISCUSSION TITLE
Issue with the google app

### DISCUSSION THREAD

**Discussion by hallvalerie** [potmat/homebridge-google-nest-sdm#164] [Q&A]
**Have you read the [FAQ](https://github.com/potmat/homebridge-google-nest-sdm#faq)?**

Yes

**Have you searched [existing issues](https://github.com/potmat/homebridge-google-nest-sdm/issues?q=is%3Aissue)?**

Yes

**Have you searched [existing discussions](https://github.com/potmat/homebridge-google-nest-sdm/discussions?discussions_q=)?**

Yes

**Describe the bug**
issue is with the developers.google site

**Attach Logs**
N/a

**Screenshots**
see photo
![Screenshot 2024-03-21 171850](https://github.com/potmat/homebridge-google-nest-sdm/assets/164425249/4f11c911-7d65-44fc-9983-fac67c271894)

**Environment**
google

**Additional context**
Issue occurs at this step:
https://developers.google.com/nest/device-access/authorize#get_an_access_token

Even when following your instructions by using this url instead:
https://nestservices.google.com/partnerconnections/project-id/auth?redirect_uri=https://www.google.com&access_type=offline&prompt=consent&client_id=oauth2-client-id&response_type=code&scope=https://www.googleapis.com/auth/sdm.service+https://www.googleapis.com/auth/pubsub

Please let me know when this step is fixed, so I can add my devices into your plugin.

Thank you

**Comment by litex2x** [comment]
Did you replace project-id and oauth2-client-id in that url before you opened it?

**Comment by potmat** [comment]
Converting to discussion as this is not an issue with the plugin itself.
